packages/exporter-trace-otlp-proto/test/OTLPTraceExporter.test.ts

```diff
@@ -261,7 +277,86 @@ describe('OTLPTraceExporter - node with proto over http', () => {
         buff = Buffer.concat([buff, chunk]);
       });
 
+      const clock = sinon.useFakeTimers();
       collectorExporter.export(spans, () => { });
+      clock.tick(200);
+      clock.restore();
+    });
+  });
+  describe('export - real http request destroyed before response received', () => {
+    const server = http.createServer((_, res) => {
+      setTimeout(() => {
+        res.statusCode = 200;
+        res.end();
+      }, 200);
+    });
+    beforeEach(() => {
+      server.listen(8080);
+    });
+    afterEach(() => {
+      server.close();
```

Comment:
[`server.close()` is async](https://nodejs.org/dist/latest-v16.x/docs/api/http.html#serverclosecallback) so maybe use
```js
afterEach((done) => {
      server.close(done);
}
```
```